### PR TITLE
configure micro-airport-location in Kubernetes

### PR DESCRIPTION
**Pull request by cichan02**
## PR-Codex overview
This PR adds Kubernetes deployment and service configurations for Redis and MongoDB, and a load balancer for the airport location service.

### Detailed summary
- Added Kubernetes ConfigMaps for Redis and MongoDB
- Added Kubernetes Secrets for MongoDB authentication
- Added Kubernetes Services for Redis, MongoDB, and airport location
- Added Kubernetes Deployments for Redis, MongoDB, and airport location
- Added Kubernetes StatefulSet for MongoDB with persistent volume claims

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: micro-airport-location/redis-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  template:
+    metadata:
+      labels:
+        app: redis
+        version: v1
+    spec:
+      containers:
+        - name: redis
+          image: redis:6.0
+          imagePullPolicy: IfNotPresent
```
Comment (h1alexbel) [inline micro-airport-location/redis-deployment.yml:24]: @cichan02 should be `Always`
we expect our pod to be crashed each time the container not found in the registry

File: micro-airport-location/mongo-statefulset.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  template:
+    metadata:
+      labels:
+        app: mongo
+        version: v1
+    spec:
+      containers:
+        - name: mongo
+          image: mongo:5.0.16
+          imagePullPolicy: IfNotPresent
```
Comment (h1alexbel) [inline micro-airport-location/mongo-statefulset.yml:27]: @cichan02 should be Always
we expect our pod to be crashed each time the container is not found in the registry

**Review by h1alexbel — CHANGES_REQUESTED** [review]

**Comment by cichan02** [comment]
Injected Semantic Versioning

**Review by h1alexbel — APPROVED** [review]

**Comment by h1alexbel** [comment]
@cichan02 nice, *ensure*

**Pull request merged**